packages/web-app-files/src/views/shares/SharedWithMe.vue

```diff
@@ -172,7 +171,7 @@ export default defineComponent({
     }
 
     const shareTypes = computed(() => {
-      const uniqueShareTypes = uniq(unref(currentItems).map((i) => i.share?.shareType))
+      const uniqueShareTypes = uniq(unref(storeItems).map((i) => i.share?.shareType))
```

Comment:
Why? This is more stable and recomputes less, but then it shows share types for potentially hidden shares, no?

Reply:
Ah, ok, lesser evil then. Thanks for clarification